pkg/gardener/shoot/hyperscaler/aws/zones.go

```diff
@@ -1,14 +1,19 @@
 package aws
 
 import (
+	"github.com/pkg/errors"
 	"math/big"
 	"net/netip"
 
 	"github.com/gardener/gardener-extension-provider-aws/pkg/apis/aws/v1alpha1"
 )
 
-const workersBits = 3
-const lastBitNumber = 31
+const (
+	workersBits      = 3
```

Comment:
Could you elaborate on the meaning of this const? Why 3?

Reply:
It is really hard to explain. I wish to ask original developer of this function. I suppose the number of bits is 3 to code max 8 number of generated subnets. That's why original prefix is increased by this number. Maybe we can rename it into `subNetworkBitsSize` for clarity.